Source/WebCore/page/DeprecatedGlobalSettings.h

```diff
@@ -126,6 +126,11 @@ class DeprecatedGlobalSettings {
     static bool usesWebContentRestrictionsForFilter() { return singleton().m_usesWebContentRestrictionsForFilter; };
 #endif
 
+#if ENABLE(WIRELESS_PLAYBACK_MEDIA_PLAYER)
+    static void setWirelessPlaybackMediaPlayerEnabled(bool isEnabled) { singleton().m_wirelessPlaybackMediaPlayerEnabled = isEnabled; }
```

Comment:
We probably can do without those, and use plain settings instead with a combination of MediaStrategy/LoadOptions

Reply:
I will look into that in a follow-up.